common/plugins/com.liferay.ide.core/src/com/liferay/ide/core/util/PropertiesUtil.java

```diff
@@ -515,7 +516,7 @@ public void startElement( String uri, String localName, String qName, Attributes
         final ILiferayProject lrproject = LiferayCore.create( proj );
         final IFolder[] srcFolders = lrproject.getSourceFolders();
 
-        if( srcFolders.length < 1 )
+        if( srcFolders == null || srcFolders.length < 1 )
```

Comment:
There is a util in CoreUtil.isNullOrEmpty() can you switch this code to use that?  thanks.